### PR TITLE
Create rmt-client-setup-res

### PR DESCRIPTION
**Pull request by zpetrova**
rmt-client-setup script asssumes SUSEConnect and zypper are already installed on the system. This is not true for RHEL8, so this script installs there packages and then downloads and runs rmt-client-setup script.

### CODE REVIEW

File: public/tools/rmt-client-setup-rhel8 [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+if [ `id -u` != 0 ]; then
+    echo "You must be root. Abort."
+    exit 1;
+fi
+
+rpm --import ${REGURL}/repo/SUSE/Updates/RES/8/x86_64/update/repodata/repomd.xml.key
+dnf config-manager --add-repo ${REGURL}/repo/SUSE/Updates/RES/8/x86_64/update
+dnf config-manager --add-repo ${REGURL}/repo/SUSE/Updates/RES-AS/8/x86_64/update
+dnf install SUSEConnect sles_es-release librepo-1.9.2
```
Comment (josegomezr) [inline public/tools/rmt-client-setup-rhel8]: lgtm :+1:

**Review by josegomezr — APPROVED** [review]
lgtm :+1:

File: public/tools/rmt-client-setup-res [hunk 2]
```diff
@@ -0,0 +1,44 @@
+#! /bin/sh
+
+function usage()
+{
+    cat << EOT >&2
+
+   Usage: $0 <registration URL>
```
Comment (digitaltom) [inline public/tools/rmt-client-setup-res]: Please add a line that describes what this script is good for.

File: public/tools/rmt-client-setup-res [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+if [ `id -u` != 0 ]; then
+    echo "You must be root. Abort."
+    exit 1;
+fi
+
+rpm --import ${REGURL}/repo/SUSE/Updates/RES/8/x86_64/update/repodata/repomd.xml.key
+dnf config-manager --add-repo ${REGURL}/repo/SUSE/Updates/RES/8/x86_64/update
+dnf config-manager --add-repo ${REGURL}/repo/SUSE/Updates/RES-AS/8/x86_64/update
+dnf install SUSEConnect sles_es-release librepo-1.9.2
```
Comment (digitaltom) [inline public/tools/rmt-client-setup-res]: Should we add automatic 'yes' here?

**Review by digitaltom — APPROVED** [review]
Please see my 2 comments, other than that works good!

**Comment by digitaltom** [comment]
@zpetrova I guess we also need changes to the .spec file to get that file deployed on the rmt installation.

**Pull request merged**